modules/auxiliary/scanner/http/redoc_exposed.rb

```diff
@@ -0,0 +1,76 @@
+require 'msf/core'
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Scanner
+  include Msf::Auxiliary::Report
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'ReDoc API Docs UI Exposed',
+        'Description' => %q{
+          Detects publicly exposed ReDoc API documentation pages which may reveal API surface,
+          endpoints, request/response models, and other implementation details useful to attackers.
+        },
+        'Author' => [
+          'Hamza Sahin <hmzshn61@gmail.com>'
+        ],
+        'License' => MSF_LICENSE,
+        'References' => [
+          [ 'URL', 'https://redocly.com/docs/redoc/' ]
+        ],
+        'Actions' => [ [ 'Scan', { 'Description' => 'Scan for exposed ReDoc UI' } ] ],
+        'DefaultAction' => 'Scan'
+      )
+    )
+
+    register_advanced_options([
+      OptString.new('REDOC_PATHS', [ true, 'Comma-separated paths to probe', '/redoc,/docs,/api/docs,/openapi,/redoc/' ])
+    ])
+  end
+
+  # Returns :yes if the path looks like a ReDoc page, else :no
+  def check_path(path)
+    res = send_request_cgi({ 'method' => 'GET', 'uri' => normalize_uri(path) }, 10)
```

Comment:
No need to add timeout here:

```suggestion
    res = send_request_cgi({ 'method' => 'GET', 'uri' => normalize_uri(path) })
```